app/App/Commands/Dump.hs

```diff
@@ -172,4 +178,14 @@ runDump opt = do
               liftIO $ IO.hPutStr hOut $ (\c -> if isPrint c then c else '.') <$> LBSC.unpack bs
               liftIO $ IO.hPutStrLn hOut ""
 
-  where magic = AP.string "seg:" *> (BS.pack <$> many AP.anyWord8) AP.<?> "\"seg:????\""
+  where
+    magic = AP.string "seg:" *> (BS.pack <$> many AP.anyWord8) AP.<?> "\"seg:????\""
+    bitsToW32s :: Int -> (Int, Word64) -> [Word32] -> [Word32]
+    bitsToW32s bp (i, w64) acc =
```

Comment:
Maybe `word64ToList` would be useful to reuse?

Reply:
```λ> word64ToList 32 0x003300 []
[0.0.8.8,0.0.8.9,0.0.8.12,0.0.8.13]
it :: [HaskellWorks.Data.Network.Ip.Type.Ipv4Address]
(0.18 secs, 1,079,528 bytes)
λ> bitsToW32s 0 (32, 0x003300) []
[0.0.8.50,0.0.8.51,0.0.8.54,0.0.8.55]
it :: [HaskellWorks.Data.Network.Ip.Type.Ipv4Address]
(0.02 secs, 1,105,296 bytes)
λ>
```